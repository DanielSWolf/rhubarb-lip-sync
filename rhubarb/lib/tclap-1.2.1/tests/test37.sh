#!/bin/sh

# success
../examples/test7 -n homer 2 -n marge 1 3 > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test37.out; then
	exit 0
else
	exit 1
fi

