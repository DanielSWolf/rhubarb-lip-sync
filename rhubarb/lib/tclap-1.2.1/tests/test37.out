Got num 2
Got num 1
Got num 3
Got name homer
Got name marge
